src/python/graplctl/graplctl.Dockerfile

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
not a must-have, but a nice-to-have-but-feel-free-to-ignore:

could you add a comment on why a dockerized graplctl is needed? (I *believe* it's so e2e can invoke stuff with it)

Reply:
Yeah, we need a container with graplctl installed in it so we can execute the provisioner lambda in localstack and upload the various analyzers and test datasets while running the e2e tests locally.

Reply:
This effectively serves as an integration test of those `graplctl` commands. Unfortunately we can't make it a "complete" integration test because for some unknown reason the e2e-test-runner lambda won't run in localstack (but the provisioner will).